src/pip/_internal/commands/cache.py

```diff
@@ -48,6 +48,11 @@ def run(self, options, args):
             "purge": self.purge_cache,
         }
 
+        if not options.cache_dir:
+            logger.error("Pip cache commands can not "
```

Comment:
I think this should be `pip` instead of `Pip` :)

Reply:
Yup. Otherwise LGTM.

```suggestion
            logger.error("pip cache commands can not "
```

Reply:
Nit: Should it be `cannot` instead of `can not` ?

Reply:
I like `can not` more, but I am open to other ideas 😄 
Also, thanks for the feedback guys! @uranusjr @gutsytechster @deveshks